### PR TITLE
Answer: 1

### PR DESCRIPTION
**Pull request by Amit775**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/angular/projection/src/app/ui/card/card.component.ts [hunk 1]
```diff
@@ -1,61 +1,48 @@
...
-  ) {}
-
-  addNewItem() {
-    if (this.type === CardType.TEACHER) {
-      this.teacherStore.addOne(randTeacher());
-    } else if (this.type === CardType.STUDENT) {
-      this.studentStore.addOne(randStudent());
-    }
-  }
+  public listItem = contentChild.required(ListItemDirective<T>, {
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.ts:45]: 😍

**Comment by tomalaforge** [comment]
Nice use of all new signal queries

**Comment by tomalaforge** [comment]
Nice way better 👍

**Comment by Amit775** [comment]
> Nice way better 👍

🤗 thanks 🙏🏽

**Pull request closed without merging**